src/python/pants/core/util_rules/archive.py

```diff
@@ -32,14 +37,36 @@ class ArchiveFormat(Enum):
     ZIP = "zip"
 
 
+class ArchiveSubsystem(Subsystem):
+    # TODO define proper scope
+    options_scope = "foobar"
+    # options_scope = GLOBAL_SCOPE # FIXME fails with  DuplicateScopeError
+    # TODO define proper help
+    help = "foobar options"
+
+    _archive_search_paths = StrListOption(
+        "--archive-search-paths",
+        default=["<PATH>"],
```

Comment:
I think the default should be the values in SEARCH_PATHS, so things continue to work as before by default...